e2e/main_test.go

```diff
@@ -94,13 +94,13 @@ func mockEnvoyVersionsServer() (*httptest.Server, error) {
 			h := r.Header.Get(k)
 			if h != v {
 				w.WriteHeader(500)
-				w.Write([]byte(moreos.Sprintf("invalid %q: %s != %s\n", k, h, v))) //nolint
+				_, _ = w.Write([]byte(moreos.Sprintf("invalid %q: %s != %s\n", k, h, v)))
```

Comment:
please revert ones like `_, _ = w` as they are worse than the trailing //nolint comment as they clutter the file. The real problem is the IDE not honoring `//nolint`.

Adding pressure to https://youtrack.jetbrains.com/issue/GO-6502 is the better way than adding debt everywhere IMHO. Please upvote that if you don't mind! Sometimes issues go stale and have resolved in JetBrains. That this isn't a "standard" is invalid because de-facto standards end up in IDE config often enough.

PS I know not everyone shares this opinion, but I do think if we weren't slaves to IDE defects, we'd agree this is worse to read. cc @llinder and @johnlanda who I think decided to be slaves in another project 😎 I just really don't want to set a precedent here where I end up intentionally making code harder to read over this.

Reply:
PS looks like "won't fix" disables voting. If someone knows someone who works at JetBrains, can you please tell them to unlock https://youtrack.jetbrains.com/issue/GO-6502? The answer there is entirely dismissive, draconian and removed from reality

Reply:
Since https://youtrack.jetbrains.com/issue/GO-6502 is locked I opened the next best https://github.com/xxpxxxxp/intellij-plugin-golangci-lint/issues/92, as at least there it is possible to upvote!